.chloggen/pkg-stanza-rm-sugared-api.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: deprecation
```

Comment:
Mostly for my understanding: isn't this also a breaking change since the `Operator` interface changes and specifically the `Logger() *zap.SugaredLogger` -> `Logger() *zap.Logger`?

Reply:
Yeah, good point.